docassemble/AssemblyLine/data/questions/al_visual.yml

```diff
@@ -43,6 +43,7 @@ default screen parts:
   pre: |
     % if get_config('debug'):
     `id: ${ user_info().question_id }`  
+    `Variable triggering this screen: ${ user_info().variable }`  
```

Comment:
We'll want to do maybe a bit more than just `user_info().variable` here, since that will just show `x.my_attr` for generic questions.

Trying to reconstruct the actual variable inline can get confusing, but this is a start:

```suggestion
    `Variable triggering this screen: ${ user_info().variable } ${("x = " + str( x.instanceName )) if defined('x') else ''} ${ ("i =" + str( i )) if defined('i') else ''}`  
```

https://github.com/SuffolkLITLab/ALKiln/issues/582 has some more details on a similar thing in Kiln.

Reply:
Not sure if this actually adds something in this context -- can you give an example of a screen where it's useful so I can better evaluate?

![image](https://user-images.githubusercontent.com/7645641/203136982-63f0fb4e-b380-43b5-895e-573652c0857d.png)

Reply:
shoot, never mind, I was wrong. I think I was thinking of some of the kiln issues, which are getting the variable names from the fields; those would look more like `x[i].name.first`. I thought `user_info.variable()` had the same issue but it doesn't, sorry.